backend/bookings/services/bookingService.js

```diff
@@ -1,23 +1,28 @@
-const { sequelize } = require("@utopia-airlines-wss/common/db");
-const { Booking } = require("@utopia-airlines-wss/common/models");
-const {  NotFoundError,  handleMutationError   } = require("@utopia-airlines-wss/common/errors");
+const {sequelize} = require("@utopia-airlines-wss/common/db");
+const {Booking} = require("@utopia-airlines-wss/common/models");
+const {NotFoundError, handleMutationError} = require("@utopia-airlines-wss/common/errors");
 
 const findBookingById = async (id, options) => {
   const booking = await Booking.findByPk(id, options);
   if (!booking) throw new NotFoundError("cannot find booking");
   return booking;
 };
 
-const replacer = (key, value) =>  key === "passengers"
+const replacer = (key, value) => key === "passengers"
   ? value.map((passenger) => passenger.toJSON("full"))
   : value;
 
-
 const bookingService = {
-  async findAllBookings({ isActive = true } = {}) {
-    const where = { isActive };
-    const bookings = await Booking.findAll({
+  async findAllBookings(
+    {isActive = true, offset = 0, limit = 10} = {}) {
+    const where = {isActive};
+    limit = limit ? Math.min(+limit, 40) : 10;
```

Comment:
Is there a reason we are setting a maximum?

Reply:
Yes, it is best practice.

Reply:
From what I can see best practices are to set a cap of 10,000 or more. The purpose is only to prevent any server instance from crashing due to running out of memory. If an authenticated user requests 1000 entries, they should be allowed to get it.

Reply:
We can decide what the cap would be.